lib/cartopy/io/srtm.py

```diff
@@ -246,29 +250,61 @@ def srtm(lon, lat):
     return SRTM3Source().single_tile(lon, lat)
 
 
-def add_shading(elevation, azimuth, altitude):
+def add_shading(located_image, azimuth=315, altitude=45, scale=111120, z=1):
     """Adds shading to SRTM elevation data, using azimuth and altitude
-    of the sun.
-
-    :type elevation: numpy.ndarray
-    :param elevation: SRTM elevation data (in meters)
+    of the sun. Function assumes x,y,z in same units, but automatically
```

Comment:
The second sentence should start on a new paragraph (one blank line between each paragraph.)